README _En.md

```diff
@@ -0,0 +1,44 @@
+# Bitcoin & Lightning Network `Francophone` Ressources HUB
+
+![banner](assets/header-cover.png)
+
+Available in 2 Versions || [English]() and [French](https://github.com/richarddushime/BLN-EduHub-Francophone/blob/main/README.md) ||
```

Comment:
The default file should be `README.md`. So, I feel like redirecting the user to the `README` file that has a language extension would be logic. If the README.md is in French, then the [English] link should redirect to `README-en.md`

Reply:
Alright Done it